app/frontend/src/features/communities/events/EventAttendees.tsx

```diff
@@ -0,0 +1,35 @@
+import { useState } from "react";
+
+import { ATTENDEES, NO_ATTENDEES } from "./constants";
+import EventAttendeesDialog from "./EventAttendeesDialog";
+import EventUsers from "./EventUsers";
+import { useEventAttendees } from "./hooks";
+
+interface EventAttendeesProps {
+  eventId: number;
+}
+
+export default function EventAttendees({ eventId }: EventAttendeesProps) {
+  const { attendees, attendeesIds, error, isLoading, hasNextPage } =
```

Comment:
How about not destructuring the query and passing that in?

Reply:
Hmm I thought about it, but it makes rendering the `EventUsers` on its own in Storybook a pain as I'd have to mock the full object `useInfiniteQuery` returns. Or I could probably use the query for real and set the data in the cache since we already wrap the stories with a query client. Maybe I'll do that 🤔

Reply:
Don't you mock the service?
Well, whatever works

Reply:
`EventUsers` take in these props and since it's purely presentational (so I can share it between attendees and organisers), it only needs static data passed in and no React Query involved. So if I pass the whole query down as a prop, I'd have to write out that whole object rather only the props it needs.